src/ConnectedVMware/docs/New-AzConnectedVMwareVMGuestAgent.md

```diff
@@ -37,27 +37,18 @@ Create GuestAgent.
 
 ## EXAMPLES
 
-### Example 1: {{ Add title here }}
+### Example 1: Enable Guest Agent on VM
 ```powershell
-{{ Add code here }}
+New-AzConnectedVMwareGuestAgent -Name "default" -VirtualMachineName "test-vm" -ResourceGroupName "test-rg" -SubscriptionId "204898ee-cd13-4332-b9d4-55ca5c25496d" -CredentialsUsername $VM_User_Name -CredentialsPassword $Secure_String_Pwd -ProvisioningAction "install"
```

Comment:
>-Name "default" [](http://example.com/codeflow?start=32&length=15)

do we still need to give name when we have default in swagger?

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/stable/2023-10-01/connectedvmware.json#L3109](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/stable/2023-10-01/connectedvmware.json#L3109)

Reply:
updated